articles/azure-functions/functions-bindings-cosmosdb-v2.md

```diff
@@ -2097,14 +2100,6 @@ Here's the F# code:
     open FSharp.Interop.Dynamic
     open Newtonsoft.Json
     open Microsoft.Extensions.Logging
-
```

Comment:
Type is never used in the example code, so either remove the type, or keep the type and use it in the code. As is, there is no need to declare it